dynawo/sources/Common/Dictionaries/DYNError_en_GB.dic

```diff
@@ -156,6 +156,7 @@ DynamicLineStatusNotSupported =           connection status (CLOSED_1 or CLOSED_
 MismatchingVariableSizes    =             the number of %1% variables and corresponding size are different in model %2% (%3% != %4%)
 ConverterWrongType          =             wrong converter type for converter %1%
 AttemptToPropagateBeforeMerge        =    discrete variables connectors should be merged before propagating information to model
+ZValueIsNaN                 =             z value is NaN in discrete variables buffer at index %1%
```

Comment:
I would propose "discrete variable with index %1% has a NaN value"

Reply:
I agree, it's more neat